### PR TITLE
HTML and CSS Project 2: HTML Forms

### PR DESCRIPTION
**Pull request by ricardomonte**
In this project:

- We structure the html using:

> Semantic

> Normalizer

> CSS stylesheet

- We also create the style using:

> CSS Flexbox

> CSS Grid

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+              Email address
+              <input type="email" name="email" required />
+            </label>
+            <label class="label-phone lbl-position">
+              Phone (recommended)
+              <input type="tel" name="phone" />
+            </label>
+            <span>Standard call, messging or data rates may apply</span>
+            <div class="container-password">
+              <div class="label-password lbl-position" tabindex="0">
```
Comment (kevcap) [inline index.html]: Remember to use [semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp). You guys forgot to use the correct tag here. Kindly fix that.

**Review by kevcap — CHANGES_REQUESTED** [review]
# [STATUS] CHANGES REQUIRED♻️

 
### Hi team, you guys did a very good job so far! 💪
Amazing use of semantic HTML5, code looks also very clean and professional. 👏👌


#### I know you guys are working hard at this project, and you're almost done, there are just some required changes* that you guys have to implement in order to proceed:

[REQUIRED CHANGES]
- [x] Check usage of [semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/semantic_html_requirement.png) from Pathwright)
- [x] Remove all the unnecessary repeated CSS code ([Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/))
- [x] Add your CSS files to a style or css folder. (It will make your repo look more professional)

[OPTIONAL CHANGES]
- [x] Make your Readme file look more professional by working on its layout (add emojis and so on)

### Tips:

* In your CSS file, there's a lot of repeated rules. e.g.: Instead of having 6x `width: 100%;`, create a single class: `.width-100 { 
width: 100% 
}`   and apply it in your HTML file.
image




**Please make the required changes and resubmit for a new code review once you are ready**

 

As always, I wish you all the very best and see you both at the top🚀


👨🏻‍💻 **TSE: Kevin Alves**

 

 

📫 Have any question? Better call Kevin:

 

<a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

<sub>*We're here to help you, to ensure that you're using the best practices and getting you ready for the job market. In real-world, you'll have a lot of changes requests for your real clients, it's good to get used to it.</sub>

**Review by SpaYco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello Team! 👋 
Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 

- [x] The [HTML Validator](https://validator.w3.org/#validate_by_upload) shows an error 
image

## Optional 

- [x] your form is not well centered, please fix that

- [x] change your page's title


### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Review by MuriloRoque — APPROVED** [review]
# Status: Approved :green_circle: 

## Hello @ricardomonte and @kcotrinam92 :wave: 

## You did an amazing job by making all of the required changes! Congratulations, your project is approved :clap: :clap: :clap: 

## You may merge to the master branch. Good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**